Authenticator/Source/TableDiff.swift

```diff
@@ -33,27 +33,45 @@ enum Change {
     case Move(fromIndex: Int, toIndex: Int)
 }
 
-func diff<Row>(from oldRows: [Row], to newRows: [Row], comparator isSameRow: (Row, Row) -> Bool)
-    -> [Change]
+func diff<Row>(from oldRows: [Row], to newRows: [Row],
+    comparator rowsHaveSameIdentity: (Row, Row) -> Bool) -> [Change]
 {
-    return changes(from: ArraySlice(oldRows), to: ArraySlice(newRows), comparator: isSameRow)
+    return changes(from: ArraySlice(oldRows), to: ArraySlice(newRows),
+        comparator: rowsHaveSameIdentity, equation: { (old, new) in false })
+}
+
+func diff<Row: Equatable>(from oldRows: [Row], to newRows: [Row],
+    comparator rowsHaveSameIdentity: (Row, Row) -> Bool) -> [Change]
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)